edx_oauth2_provider/tests/test_oidc.py

```diff
@@ -130,7 +132,7 @@ def assertIncludedClaims(self, claims, expected_scope=None, expected_claims=None
             if scope == 'email':
                 expected_claims.extend(['email'])
 
-        self.assertItemsEqual(claims.keys(), expected_claims)
```

Comment:
This was changed in python3. Used sixfix that.

Reply:
Rather than doing `from six import assertCountEqual` use the pattern python-modernize uses.  `import six` and then call `six.assertCountEqual`  This makes it easier to find all the six related calls later when we want to just move to python3 only support.,